mkdocs.yml

```diff
@@ -22,8 +22,6 @@ nav:
   - detector-geometry.md
   - Components:
     - components/index.md
-    - components/scans.md
-    - components/pulse-patterns.md
```

Comment:
I'd prefer if all pages were listed in the index. In general I dislike having disembodied pages because otherwise navigating to them requires that someone knows the right link to click on the right page (e.g. I always get confused when trying to find the xwiz docs).